mmv1/third_party/terraform/services/discoveryengine/resource_discovery_engine_chat_engine_test.go

```diff
@@ -0,0 +1,115 @@
+package discoveryengine_test
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"testing"
+)
+
+func TestAccDiscoveryEngineChatEngine_discoveryengineChatengineBasicExample_update(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDiscoveryEngineChatEngine_discoveryengineChatengineBasicExample_basic(context),
+			},
+			{
+				ResourceName:            "google_discovery_engine_chat_engine.primary",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"chat_engine_config"},
+			},
+			{
+				Config: testAccDiscoveryEngineChatEngine_discoveryengineChatengineBasicExample_update(context),
+			},
+			{
+				ResourceName:            "google_discovery_engine_chat_engine.primary",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"chat_engine_config"},
+			},
+		},
+	})
+}
+
+func testAccDiscoveryEngineChatEngine_discoveryengineChatengineBasicExample_basic(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+	resource "google_discovery_engine_data_store" "test_data_store" {
+		location                    = "global"
```

Comment:
could either the basic test or the update test use a non-global location? Just in case there's some edge case related to that distinction.

Reply:
added another step to change location to eu (forces replacement on all resources)